src/main/java/com/redhat/foreman/ForemanAPI.java

```diff
@@ -103,7 +103,12 @@ public JsonNode reserveHost(String hostname) {
      * @return string to be used for reserving.
      */
     private String getReserveReason() {
-        return "Reserved for " + Jenkins.getInstance().getRootUrl();
+        String url = "unknown";
+        Jenkins instance = Jenkins.getInstance();
+        if (instance != null) {
+            url = instance.getRootUrl();
```

Comment:
The code has this:

`if(SystemProperties.getBoolean(Jenkins.class.getName()+".enableExceptionOnNullInstance")){`

which is not set by default...so null is possible.

We should be using getActiveInstance(). I will amend.